src/log/CommandLogger.ts

```diff
@@ -0,0 +1,26 @@
+import SFPLogger, {COLOR_HEADER, COLOR_KEY_MESSAGE, LoggerLevel} from "@flxbl-io/sfp-logger";
+
+export default class CommandLogger {
+
+    private loggerLevel:LoggerLevel;
+    constructor() {
+        this.loggerLevel = LoggerLevel.INFO;
```

Comment:
Probably move to src/core/display and rename it CommandHeaderDisplayer or something that matches it

Reply:
Probably also take a logger as an input here , so the logger can take appropriate action as rquired, For instance if  i pass VoidLogger .. this will not get printed